src/style.css

```diff
@@ -0,0 +1,199 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 5%);
+}
+/* #endregion */
+/* #region Nav */
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.logo_img {
+  height: 100%;
+  width: 100%;
+}
+
+.main_nav {
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav_item {
+  margin-left: 20px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_item__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav_item__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background-color: #00acdc;
+  content: "";
+}
+/* #endregion */
+
+.card {
+  box-sizing: border-box;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__photo {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin: 0 0 4px;
+  color: #060b35;
+}
+
+.card__name-code {
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.rate {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.price__price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.buy {
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+
+}
+
+.buy:hover {
+  background-color: #fff;
+  border-color: #00acdc;
```

Comment:
`border-color` is already there in default `.buy` so it will inherit here , you can delete it from here.